src/coreclr/src/tools/crossgen2/ILCompiler.ReadyToRun/Compiler/DependencyAnalysis/ReadyToRun/MethodWithGCInfo.cs

```diff
@@ -219,7 +219,9 @@ public virtual void AppendMangledName(NameMangler nameMangler, Utf8StringBuilder
         protected override string GetName(NodeFactory factory)
         {
             Utf8StringBuilder sb = new Utf8StringBuilder();
+            sb.Append($@"MethodWithGCInfo(");
             AppendMangledName(factory.NameMangler, sb);
+            sb.Append($@")");
```

Comment:
I wanted to nitpick about the useless `$@` prefix, but I see this is consistent with useless `$@` prefixes in a bunch of other places so I'll just cringe silently instead.

This line has a nice combo of characters one needs to hold `Shift` for and the readability is competitive with Perl.

Reply:
Lol, you don't need to cringe silently. This is ridiculous, and an easy fix which I've done, although I'd say the readability isn't quite *that* bad.  :)